senior_teacher.rb

```diff
@@ -1,50 +1,19 @@
-class SeniorTeacher
-  attr_reader :age, :salary, :phase, :performance_rating, :target_raise
-  attr_accessor :name
+require_relative './teacher/'
+require_relative './high_five/'
 
-  def initialize(options={})
-    @phase = 3
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-    @target_raise = 1000
-  end
+class SeniorTeacher < Teacher
+  include HighFive
```

Comment:
Glad that you used a module, however this is common to all.  So where should you ideally put this?